src/engraving/utests/barline_tests.cpp

```diff
@@ -344,65 +344,83 @@ TEST_F(BarlineTests, barline179726)
     Measure* m = score->firstMeasure();
 
     // drop NORMAL onto initial START_REPEAT barline will remove that START_REPEAT
-    dropNormalBarline(m->findSegment(SegmentType::StartRepeatBarLine, m->tick())->elementAt(0));
```

Comment:
no changes to what's actually being tested here, just ensuring we get clear/helpful error messages if `findSegment` returns `nullptr`